test-suite-thymeleaf-fieldset/src/test/java/io/micronaut/views/fields/thymeleaf/Login.java

```diff
@@ -0,0 +1,10 @@
+package io.micronaut.views.fields.thymeleaf;
+
+import io.micronaut.core.annotation.Introspected;
+import io.micronaut.views.fields.annotations.InputPassword;
+import jakarta.validation.constraints.NotBlank;
+
+@Introspected
```

Comment:
Should this be `@Serdeable`? 🤔

Reply:
It does not need to. It needs to be `@Introspected. If you want to use it in an application using Micronaut Serialization you will do `@Serdeable` which does `Introspected` automatically.